ext/standard/tests/directory/DirectoryClass_cannot_construct.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Cannot directly instantiate Directory class.
+--FILE--
+<?php
+
+try {
+    $d = new Directory();
+    var_dump($d);
```

Comment:
in the same vein, I suggest removing the `var_dump()` here and in the cannot_serialize test - it should never be reached

Reply:
The point is to have an output in case it was able to be initialized, moreover, this was introduced in a prior commit to see the difference in behaviour.